lib/measured/arithmetic.rb

```diff
@@ -15,23 +15,25 @@ def /(other)
     arithmetic_operation(other, :/)
   end
 
-  def -@
-    self.class.new(-self.value, self.unit)
+  def coerce(other)
```

Comment:
Can you add tests for this method directly. Clearly it was not properly tested to begin with. Testing our three cases of the same class, a Numeric class, and a type error.